src/coreclr/debug/createdump/createdumpmain.cpp

```diff
@@ -126,6 +127,10 @@ int createdump_main(const int argc, const char* argv[])
             {
                 options.AppModel = AppModelType::SingleFile;
             }
+            else if (strcmp(*argv, "--nativeaot") == 0)
```

Comment:
Should this be a mode flag - or are we OK with having to maintain one flag per option? Since I don't foresee a lot of them, it's ok. However, this means that if someone has several in the list, last one wins.

Reply:
I don't think this is that big of deal.